src/operator/tensor/square_sum-inl.h

```diff
@@ -359,6 +353,24 @@ void SquareSumRspImpl(const nnvm::NodeAttrs& attrs,
   }
 }
 
+/*!
+ * \brief check the indices of ograd and input are the same.
+ */
+struct CheckSameIdxKernel {
+  template<typename IType>
+  MSHADOW_XINLINE static void Map(int i, IType* ograd_idx, IType* in_idx, int32_t* is_same) {
+    if (ograd_idx[i] != in_idx[i]){
+      *is_same = 1;
```

Comment:
The name of `is_same` looks like a `is_different` after checking the logic, or should it be assigned with `0` rather than `1` if `ograd_idx[i] != in_idx[i]`?